site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -193,4 +271,27 @@ This is particularly true for a zonal GKE control plane, which can go down tempo
 
 ## Next Steps
 
-- Continue to [Install Agones]({{< relref "../Install Agones/_index.md" >}}).
+### GKE Standard
+
+If you created a GKE Standard cluster, continue to [Install Agones]({{< relref "../Install Agones/_index.md" >}}).
+
+### GKE Autopilot
```

Comment:
Since people may think the website docs apply consistently across all releases, it might be worth making it clear that these are the instructions for Agones 1.30 (we will probably update them for 1.31 so I think it's ok to call out 1.30 explicitly).

Reply:
Ah okay, works for me